S2VX.Game/SongSelection/Containers/SongPreview.cs

```diff
@@ -15,10 +16,11 @@ public class SongPreview : CompositeDrawable {
         [Resolved]
         private ScreenStack Screens { get; set; }
 
-        public string CurSelectionPath { get; set; }                           // Set in SongSelectionScreen
-        public string StoryPath { get; set; }                                  // Set in SongSelectionScreen
-        public string AudioPath { get; set; }                                  // Set in SongSelectionScreen
-        public StorageBackedResourceStore CurLevelResourceStore { get; set; }  // Set in SongSelectionScreen
+        public string CurSelectionPath { get; set; }                           // Must be set in SongSelectionScreen
```

Comment:
Why don't you just set these in a constructor since they are required?

Reply:
Will do

Reply:
I did a quick Ctrl+F, and I think Border still has something with the `// Set in SongSelectionScreen` comment. Should update this too?

Reply:
Bumping in case you missed this

Reply:
Ah only Ctrl+F'd for "must". Thanks for catching this, my bad for missing. Updated.